torch/_tensor_docs.py

```diff
@@ -4146,6 +4146,20 @@ def callable(a, b) -> number
            size=(3, 3), nnz=1, layout=torch.sparse_coo)
 """)
 
+add_docstr_all('to_sparse_csr',
+               r"""
+to_sparse_csr() -> Tensor
+Convert a tensor to compressed row storage format. Only works with 2D tensors.
+
+Example::
+
+    >>> dense = torch.randn(5, 5)
+    >>> sparse = dense.to_sparse_csr()
+    >>> sparse._nnz()
+    25
```

Comment:
This example uses a private feature (`_nnz`) and may give a different result when executed locally (although, rarely).
```suggestion
    >>> sparse.is_sparse_csr
    True
```
? Or use `dense=torch.tensor([[1, 0, 2], [0, 3, 0]])` as an example input and use `repr(sparse)` as example output?